testing/injections/timers_disabled_shim.js

```diff
@@ -0,0 +1,13 @@
+// ts file compiled js file somehow end up with cjs format but esbuild needs esm format for shims.
+// so for now we just have this file in js.
+
+const ErrorMessage = 'Please use --timers option to enable timers. Be aware that timing methods ' + 
```

Comment:
Nit, I would be pretty specific here, something like:

Please use the --timers flag to enable shimmed timing primitives. Native node timing primitives like setTimeout or setInternal aren't supported in the pack execution sandbox environment. However, if you are using a library that relies upon them, you can use the --timers flag to build your pack with shimmed implementations that approximate the native behavior. Because of this, be aware that packs that use timing primitives may not work reliably.